codebundles/rds-mysql-conn-count/README.md

```diff
@@ -0,0 +1,95 @@
+# CodeBundle - RDS MySQL Connection Count
+
+This codebundle targets to detect and resolve an incident caused by too many sleeping connections in MySQL.
+
+- Target Service - MySQL
+- Cloud Platform - AWS/RDS
+
+## SLX
+```YAML
+statement: RDS MySql connections should be within 80% of total max connection.
+alias: RDS MySql Connections Count
+metricType: gauge
+asMeasuredBy: Score based on promethues query
+icon: Cloud
+owners:
+  - saurabh.yadav@infracloud.io
+imageURL: >-
+  https://storage.googleapis.com/runwhen-nonprod-shared-images/icons/kubernetes/resources/labeled/ns.svg
+
+```
+## SLO / Service Level Objective
+Example:
+```YAML
+codeBundle:
+  repoUrl: https://github.com/runwhen-contrib/rw-public-codecollection
+  pathToYaml: codebundles/slo-default/queries.yaml
+  ref: main
+sloSpecType: simple-mwmb
+objective: 95
+threshold: 48
+operand: lt
+```
+
+## SLI / Service Level Indicator
+```YAML
+displayUnitsLong: OK
+displayUnitsShort: ok
+locations:
+  - location-01-us-west1
+description: >-
+  Watch RDS MySql connection count
+codeBundle:
+  repoUrl: https://github.com/infracloudio/ifc-rw-codecollection
+  ref: main
+  pathToRobot: codebundles/rds-mysql-conn-count/sli.robot
+# read more about intervalStrategy here: https://docs.runwhen.com/public/runwhen-platform/feature-overview/points-on-the-map-slxs/service-level-indicators-slis/interval-strategies
+intervalStrategy: intermezzo
+intervalSeconds: 30
+configProvided:
+  - name: PROMETHEUS_HOSTNAME
+    value: >-
```

Comment:
can write that change PROMETHEUS_HOSTNAME to your endpoint and currently endpoint needs to be publicly exposed.